apps/extension/webpack/SimpleLocalizeDownloadPlugin.js

```diff
@@ -51,59 +52,90 @@ module.exports = class SimpleLocalizeDownloadPlugin {
       return
     }
 
```

Comment:
yes while working in dev mode with multilanguage environment keys provided, it re-downloads on each code change, which is very annoying and felt unintended. with this change it would only download on startup